db/db.js

```diff
@@ -11,4 +11,11 @@ if (!["production", "development"].includes(DEPLOYMENT_MODE)) {
   console.error("DEPLOYMENT_MODE must be either 'production' or 'development'");
 }
 
+export function user_info() {
+  return db("users")
+    .select(["users.id as id", "username", "balance"])
+    .join("balances", "balances.id", "=", "users.id")
+    .orderBy("balance");
```

Comment:
I'm pretty sure you have a different function called `user_info_wos` that you meant to commit, but putting a generic `user_info` function in the `db` module is good. I would remove the `orderBy` from this function and let callers add any ordering they want like so:

```js
await user_info().orderBy('balance').limit(10);
```

In general we want to avoid having application-specific functions in the `db` module, but something like `user_info` is useful in both `inventory` and `wos`

Reply:
Fixed and added the check to verify that the balances are under $-5.